package.json

```
@@ -16,7 +16,9 @@
 		"precommit": "pnpm format && pnpm lint",
 		"changeset:version": "changeset version && pnpm -r generate:version && git add --all",
 		"changeset:release": "changeset publish",
-		"start": "cd sites/kit.svelte.dev && npm run dev"
+		"start": "cd sites/kit.svelte.dev && npm run dev",
+		"changeset": "npx changeset"
```

there should be no need to change this file for documentation changes
```suggestion
		"start": "cd sites/kit.svelte.dev && npm run dev"
```